delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/organizations/model/OrganizationRejection.scala

```diff
@@ -77,6 +77,15 @@ object OrganizationRejection {
   final case class OrganizationIsDeprecated(label: Label)
       extends OrganizationRejection(s"Organization '$label' is deprecated.")
 
+  /**
+    * Signals an attempt to update/deprecate an organization that is already deprecated.
```

Comment:
...an organization that is already active?

Reply:
I'll fix this in another PR, good spot